packages/TorneloScoresheet/src/components/PrimaryText/PrimaryText.tsx

```diff
@@ -36,10 +37,12 @@ const PrimaryText: React.FC<PrimaryTextProps> = ({
   style,
   size,
   children,
+  numberOfLines,
   align,
 }) => {
   return (
     <Text
+      numberOfLines={numberOfLines}
```

Comment:
Would it maybe make sense to default the number of lines to 1 if possible? Because I imagine that's what it will be in most places

Reply:
I don't think it would make sense to make it default, the number of lines argument truncates your text to fit on one line, I think if that is your target behaviour you should be setting it in the component, if it's default it might surprise us in some places